src/odemis/driver/test/technolution_test.py

```diff
@@ -0,0 +1,1460 @@
+        min_scanOffset = self.MirrorDescanner.scanOffset.range[0][0]
+        max_scanOffset = self.MirrorDescanner.scanOffset.range[1][0]
+
+        # Check if small scanOffset values are allowed
+        self.MirrorDescanner.scanOffset.value = (0.1 * max_scanOffset, 0.1 * max_scanOffset)
+        self.assertEqual(self.MirrorDescanner.scanOffset.value, (0.1 * max_scanOffset, 0.1 * max_scanOffset))
+
+        # Check if big scanOffset values are allowed
+        self.MirrorDescanner.scanOffset.value = (0.9 * max_scanOffset, 0.9 * max_scanOffset)
+        self.assertEqual(self.MirrorDescanner.scanOffset.value, (0.9 * max_scanOffset, 0.9 * max_scanOffset))
+
+        # Check if VA refuses to set limits outside allowed range
+        with self.assertRaises(IndexError):
+            self.MirrorDescanner.scanOffset.value = (1.2 * max_scanOffset, 1.2 * max_scanOffset)
+        self.assertEqual(self.MirrorDescanner.scanOffset.value, (0.9 * max_scanOffset, 0.9 * max_scanOffset))
+
+        with self.assertRaises(IndexError):
+            self.MirrorDescanner.scanOffset.value = (1.2 * min_scanOffset, 1.2 * min_scanOffset)
+        self.assertEqual(self.MirrorDescanner.scanOffset.value, (0.9 * max_scanOffset, 0.9 * max_scanOffset))
+
+    def test_scanGain_VA(self):
+        min_scanGain = self.MirrorDescanner.scanGain.range[0][0]
+        max_scanGain = self.MirrorDescanner.scanGain.range[1][0]
+
+        # Check if small scanGain values are allowed
+        self.MirrorDescanner.scanGain.value = (0.1 * max_scanGain, 0.1 * max_scanGain)
+        self.assertEqual(self.MirrorDescanner.scanGain.value, (0.1 * max_scanGain, 0.1 * max_scanGain))
+
+        # Check if big scanGain values are allowed
+        self.MirrorDescanner.scanGain.value = (0.9 * max_scanGain, 0.9 * max_scanGain)
+        self.assertEqual(self.MirrorDescanner.scanGain.value, (0.9 * max_scanGain, 0.9 * max_scanGain))
+
+        # Check if VA refuses to set limits outside allowed range
+        with self.assertRaises(IndexError):
+            self.MirrorDescanner.scanGain.value = (1.2 * max_scanGain, 1.2 * max_scanGain)
+        self.assertEqual(self.MirrorDescanner.scanGain.value, (0.9 * max_scanGain, 0.9 * max_scanGain))
+
+        with self.assertRaises(IndexError):
+            self.MirrorDescanner.scanGain.value = (1.2 * min_scanGain, 1.2 * min_scanGain)
+        self.assertEqual(self.MirrorDescanner.scanGain.value, (0.9 * max_scanGain, 0.9 * max_scanGain))
+
+    def test_getXAcqSetpoints(self):
+        """
+        After each change the same properties of the testcases are checked, first the length (total number of setpoints)
+        then the expected range of the values are checked. The maximum is not necessarily reached because this is
+        dependent on the descan_period/dwell_time
+        """
+        descanner = self.MirrorDescanner
+        scanner = self.EBeamScanner
+        mppc = self.MPPC
+        # Change values such that it is easy to follow the calculation
```

Comment:
How are these values "easy"? Are they a multiple of something and that is why they are "easy" or how did you choose them?